### PR TITLE
Enhancement: Bug Details Page: 6

### PR DESCRIPTION
**Pull request by CodeWithBishal**
The Upper part of the page has been updated to match the design of sentry issue page, in my next PR I will update the design of the marked area
Screenshot 2024-05-18 at 4 15 12 PM

### CODE REVIEW

File: website/templates/issue3.html [hunk 1]
```diff
@@ -449,4 +468,5 @@
         });
     });
 </script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/flowbite/2.3.0/flowbite.min.js"></script>
```
[the remark this answers is not in the document]
Comment (CodeWithBishal) [reply website/templates/issue3.html]: FIXED

**Comment by CodeWithBishal** [comment]
@DonnieBLT this is my next PR, I have added tooltip to the buttons

Screenshot 2024-05-20 at 7 34 49 PM

**Comment by CodeWithBishal** [comment]
Um okay, though that might mess up the design but I will get it ready by my 8th revision. This is the 6th revision, 7th is ready, as soon as you merge 6, I will create a PR for 7 then 8 with the specified changes.

Design as of 7th revision
Screenshot 2024-05-20 at 7 41 50 PM

**Comment by CodeWithBishal** [comment]
@DonnieBLT

**Review by DonnieBLT — APPROVED** [review]

**Pull request merged**